alternates/fakenews-gambling-porn-social/hosts

```diff
@@ -2354,7 +2354,7 @@ ff02::3 ip6-allhosts
 # Blocked: 1,046 domains
 # Only include advertisers in Vietnam
 # Home: https://bigdargon.github.io/hostsVN/
-# Github: https://github.com/bigdargon/hostsVN/
+# GitHub: https://github.com/bigdargon/hostsVN/
```

Comment:
This repository do not change the comments from upstream. That means, that this line stay as it is. 

If you really want to change that, please report to upstream:  https://github.com/bigdargon/hostsVN/

Reply:
The repository hasn't changed, it's the text before that that comes from the template that changed.

Reply:
@XhmikosR That line does not come from our template.

Reply:
OK, I will revert this one. See my comment below. Because I'm not sure what comes from where, I initially replaced every instance.